abz/acousticbrainz.py

```diff
@@ -67,6 +67,12 @@ def is_processed(filepath):
     else:
         return False
 
+def get_processed_status(filepath):
```

Comment:
There should be an additional blank line above here: https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
@Freso I have added a blank line here :https://github.com/MTG/acousticbrainz-client/pull/46/commits/a252727f5056946f097c1a94b52a323658a6aa18